cmd/restore/restore.go

```diff
@@ -77,8 +74,8 @@ func (c *CmdOpts) restore(path string) error {
 		logger.Fatal("k0s seems to be running! k0s must be down during the restore operation.")
 	}
 
-	if !file.Exists(path) {
-		return fmt.Errorf("given file %s does not exist", path)
+	if !file.Exists(archivePath) {
+		return fmt.Errorf("given file %s does not exist", archivePath)
```

Comment:
Just a general thought, the `mgr.RunRestore` will return `fmt.Errorf("failed to unpack backup archive `%s`: %v", archivePath, err)` from the first row if the file doesn't exist (it should probably use `%w` to wrap it so it passes as `os.ErrNotExist`). Checking for file existence is often unnecessary.